server/lib/host-metrics.js

```diff
@@ -266,6 +267,49 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(
+  host,
+  { startDate, endDate, collectiveIds = null, timeUnit } = {},
+) {
+  if (!collectiveIds) {
+    const hostedCollectives = await host.getHostedCollectives();
+    collectiveIds = hostedCollectives.map(c => c.id);
+    collectiveIds.push(host.id);
+  }
+
+  const results = await sequelize.query(
+    `SELECT SUM(t1."amountInHostCurrency") as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
```

Comment:
```suggestion
    `SELECT
      SUM(COALESCE("amountInHostCurrency", 0)) +
        SUM(COALESCE("platformFeeInHostCurrency", 0)) +
        SUM(COALESCE("hostFeeInHostCurrency", 0)) +
        SUM(COALESCE("paymentProcessorFeeInHostCurrency", 0)) +
        SUM(COALESCE("taxAmount" * "hostCurrencyFxRate", 0)) as "_amount",
      t1."hostCurrency" as "_currency",
      DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
```

Reply:
Please compute as "netAmountInHostCurrency".

Reply:
I've done the above change. 

> Please compute as "netAmountInHostCurrency".

Do you mean changing the name as well to `netAmountInHostCurrency` instead of `_amount`? This will have a problem as the `convertCurrencyForTimeSeries` function relies on the `_amount` as the variable name. 🤔 

https://github.com/opencollective/opencollective-api/blob/d3c341a90cc710d99c0a3ab76d3d23a14a2d2f22/server/lib/host-metrics.js#L37

Reply:
No, I didn't mean this.